### ISSUE TITLE
Syncing tons of tiny files taking a long time.

### ISSUE THREAD

**Issue by Zerophase** [vitalif/grive2#82]
I'm doing a first time sync of my drive to my desktop, I have a copy of Unreal up there that merges Nvidia, and other forks of the project into a more recent version of Unreal.  Most of the files being downloaded are 80kb or so.  

In four hours approximately 9GB have been downloaded. Download speeds are varying between 200 KiB/s to short blips up to 9MiB/s. I believe the download speeds ramp up on the occasional large file.

**Comment by vitalif** [comment]
Yeah grive downloads files one by one... of course it's pretty slow over http. Does official drive client do it faster?

**Comment by Zerophase** [comment]
I'll have to check, after this finishes downloading.  Last time I used the official drive client I remember it downloading 17 gigs in a couple hours. I'll let you know.

**Comment by vitalif** [comment]
Did you check the speed?

**Comment by Zerophase** [comment]
I think Google Drive on Windows was a bit faster, but still took awhile.

**Comment by j0nn0** [comment]
I believe this is because Google Drive downloads one file at a time, not in
a constant stream, so there is a pause between each file. Solution might be
to arc or zip the files. The time it takes to archive would be offset by
the time saved.

**Comment by rautamiekka** [comment]
Drive on Window$ is pretty fast uploading, considering, although it does take a while to scan loads of loads of files.

ARC was never used that much, and it loses easily to 7-Zip/XZ using LZMA2.
